webrtc.html

```diff
@@ -3273,10 +3273,13 @@ <h2>Methods</h2>
                 promise returned <code><a>setRemoteDescription</a></code> is <a>resolved</a>.
                 If <a href="#dfn-validate-the-identity">identity validation</a> fails,
                 then the promise returned by <code><a>setRemoteDescription</a></code>
-                is <a>rejected</a>.</p>
+                MUST be <a>rejected</a>.</p>
                 <p>If there is no <a>target peer identity</a>, then
-                <code>setRemoteDescription</code> does not await the completion
-                of identity validation.</p>
+                <code>setRemoteDescription</code> MUST not await the completion
+                of identity validation. Whether
+                <a href="#dfn-validate-the-identity">identity validation</a> fails
+                or not MUST NOT influence whether the promise returned by
+                <code><a>setRemoteDescription</a></code> is rejected.</p>
```

Comment:
These two sentences seem to directly contradict each other?
"If identity validation fails, then the promise returned by setRemoteDescription MUST be rejected."
"Whether identity validation fails or not MUST NOT influence whether the promise returned by setRemoteDescription is rejected."

Reply:
I think that "If there is no target peer identity" on line 3277 is intended as the "else" branch of "If a target peer identity is set" on line 3271, leaving the seemingly conflicting statements on two different branches.

Is there a better way to format this?